cfgov/v1/wagtail_hooks.py

```diff
@@ -67,15 +71,26 @@ def configure_page_revision(page, is_sharing, is_live):
 
 def flush_akamai(page, is_live):
     if is_live and settings.ENABLE_AKAMAI_CACHE_PURGE:
```

Comment:
So this bug predates your PR and I'm not sure if you want to address it here. But note that in our settings, [we try to read](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L503) this `ENABLE_AKAMAI_CACHE_PURGE` from the environment, and in the `.env_SAMPLE` [the placeholder says](https://github.com/cfpb/cfgov-refresh/blob/master/.env_SAMPLE#L38):

```sh
#export ENABLE_AKAMAI_CACHE_PURGE=<boolean_enable_purging_akamai_cache>
```

This is actually tricky, though, because if you define it as `False`, it will be read in as a string, and the above check will pass:

```sh
$ ENABLE_AKAMAI_CACHE_PURGE=False python
>>> import os
>>> if os.environ.get('ENABLE_AKAMAI_CACHE_PURGE'): print('uhoh')
... 
uhoh
```

I'm not sure what the easiest thing to do here would be, maybe just change the `.env_SAMPLE` to remove the "boolean" reference and just say "only set this to enable caching".